internal/webhook/external/receiver.go

```diff
@@ -155,33 +155,46 @@ func getDetails(
 	receiverName string,
 	secretValues ...string,
 ) (kargoapi.WebhookReceiverDetails, error) {
```

Comment:
This function is updated to find any path included in the baseURL and use it as part of the path we are building. We need to do this or else we have problems when the API server Ingress routes /webhooks to the external webhooks server.